src/steamcompmgr.cpp

```diff
@@ -2072,9 +2072,15 @@ paint_all(bool async)
 						char cmd[4096];
 						sprintf(cmd, "ffmpeg -f rawvideo -pixel_format nv12 -video_size %dx%d -i %s %s_encoded.png", pCaptureTexture->width(), pCaptureTexture->height(), pTimeBuffer, pTimeBuffer);
 
-						system(cmd);
-
-						xwm_log.infof("Screenshot saved to %s", pTimeBuffer);
+						int ret = system(cmd);
+
+						/* Above call may fail, ffmpeg returns 0 on success */
+						if (ret)
+							xwm_log.infof("Ffmpeg call return status %i", ret);
+							xwm_log.errorf( "Failed to save screenshot to %s", pTimeBuffer );
+						else {
+							xwm_log.infof("Screenshot saved to %s", pTimeBuffer);
+						}
```

Comment:
Missing braces on the if.

Reply:
Yes... I'm an idiot...

Reply:
We all make mistakes :)

Btw, please squash the two commits into one. There's no reason to have a broken commit and a fix to it in the repo if instead it's possible to just have one proper commit.

Reply:
Sure. I'll get to it once I get home. Still you can squash merge it here if you like too.